### PR TITLE
Added decryption-keys-path functionality

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Comment by lumjjb** [comment]
@stefanberger Ready for review!

File: script/tests/test_encryption.sh [hunk 1]
```diff
@@ -565,6 +610,69 @@ testJWE() {
 	$CTR images rm --sync ${ALPINE_DEC} ${ALPINE_ENC} &>/dev/null
 }
 
+testLocalKeys() {
+        createJWEKeys
+
+	echo "Testing JWE type of encryption with local unpack keys"
```
Comment (AkihiroSuda) [inline script/tests/test_encryption.sh]: shfmt?

Comment (lumjjb) [reply script/tests/test_encryption.sh]: good call, slipped my mind, I will add a new PR to add this to travis test as well.

File: cmd/ctd-decoder/main.go [hunk 2]
```diff
@@ -29,20 +30,51 @@ import (
...
+		},
+  }
+  app.Run(os.Args)
+}
+
+func run (ctx *cli.Context) error {
+	if err := decrypt(ctx); err != nil {
 		fmt.Fprintf(os.Stderr, "%s\n", err)
-		os.Exit(1)
+        os.Exit(1)
```
Comment (stefanberger) [inline cmd/ctd-decoder/main.go]: Format?

File: cmd/ctd-decoder/enc_helpers.go [hunk 3]
```diff
@@ -60,13 +78,13 @@ func getDecryptionKeys(keysPath string) (encconfig.CryptoConfig, error) {
...
-    return cc.DecryptConfig
+	cc1 := encconfig.CryptoConfig{
+		DecryptConfig: dc1,
+	}
+	cc2 := encconfig.CryptoConfig{
+		DecryptConfig: dc2,
+	}
+
+	cc := encconfig.CombineCryptoConfigs([]encconfig.CryptoConfig{cc1, cc2})
+	return cc.DecryptConfig
```
Comment (stefanberger) [inline cmd/ctd-decoder/enc_helpers.go]: Squash this into previous patch?

File: cmd/ctd-decoder/main.go [hunk 4]
```diff
@@ -27,22 +27,57 @@ import (
...
 	}
 }
 
-func decrypt() error {
+func run(ctx *cli.Context) error {
+	if err := decrypt(ctx); err != nil {
+		fmt.Fprintf(os.Stderr, "%s\n", err)
+		os.Exit(1)
+		return err
+	}
```
[the remark this answers is not in the document]
Comment (stefanberger) [reply cmd/ctd-decoder/main.go]: Any comments on this?

[the remark this answers is not in the document]
Comment (lumjjb) [reply cmd/ctd-decoder/main.go]: The error from `app.Run()` is to protect against the additional errors which may prevent problems stemming from `app.Run` itself. The documentation is not clear on this, but the examples use it like an execve, where errors are not usually checked.

**Comment by lumjjb** [comment]
rebased

**Comment by stefanberger** [comment]
LGTM

**Pull request merged**